src/vm/perfinfo.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PERFINFO_H
+#define PERFINFO_H
+
+
+#include "sstring.h"
+#include "fstream.h"
+
+class PerfInfo {
+public:
+    PerfInfo(int pid);
+    ~PerfInfo();
+    void LogImage(PEFile* pFile, WCHAR* guid); 
+
+private:
+    CFileStream* m_Stream;
+
+    const char sDelimiter = ';';
```

Comment:
You should also add a comment describing the naming scheme for the file (e.g. perfinfo-%pid.map).

---
In reply to: [58457264](https://github.com/dotnet/coreclr/pull/4068#discussion_r58457264) [](ancestors = 58457264)